demo/v3/release/output-metadata.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I don't think this or the .APK should be under version control?

Reply:
Yep, agreed. Will remove.